src/styles/main.scss

```diff
@@ -1,3 +1,48 @@
+$value:255;
+
 body {
   margin: 0;
 }
+
+html {
+  font-size: 100px;
+  font-family: Arial;
+}
+
+// .container {
+//   max-width: 1200px;
+//   display: flex;
+//   flex-wrap: wrap;
+// }
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  justify-items: stretch;
+}
+
+.anti-snake {
+  @for $i from 1 through 6 {
+    & > div:nth-child(#{$i})::after {
+      content: "#{$i}";
+      color: white;
+      justify-content: center;
+      display: flex;
+      height: 300px;
+      line-height: 300px;
+      background-color: rgb((255 - 51*($i - 1)), 0, 0);
+    }
+  }
+}
+
+@media all and (max-width: 800px) {
+  .container {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media all and (max-width: 599px) {
```

Comment:
it's possible to use only 1 media query. it's not a big problem that you're using 2, but if you want you can try to rewrite with 1 media